contentcuration/contentcuration/frontend/channelEdit/views/template/NewTopicModal.vue

```diff
@@ -0,0 +1,50 @@
+<template>
```

Comment:
Sorry, missed this earlier, but could you move these files to a new `channelEdit/move` folder? Want to make sure we don't lose the historical data if possible